### ISSUE TITLE
Browser crashes to desktop when activating Payments toggle, and selecting $10/mo option

### ISSUE THREAD

**Issue by evoorhees** [brave/browser-laptop#3672]
<img width="874" alt="screen shot 2016-09-02 at 1 55 47 pm" src="https://cloud.githubusercontent.com/assets/938106/18216667/76491aee-7115-11e6-8a43-ad8dd069155f.png">
**Did you search for similar issues before submitting this one?**
Yes

**Describe the issue you encountered:**
In Payments menu, in the Global Settings, when I click the toggle to activate Bitcoin, and then select $10/mo as my option, the browser crashes to desktop. 

I then tried simply activating Bitcoin with toggle switch, then quitting Brave (to save the setting). Then I reopened, and then selected $10, and it crashed again. 

So basically selecting $10 is crashing program 100% of the time for me.

**Expected behavior:**
Not crashing when $10 is selected

- Platform (Win7, 8, 10? macOS? Linux distro?):
MacOS

- Brave Version:
0.11.6

- Steps to reproduce:
    1. Go to Payments tab
    2. Toggle on Bitcoin 
    3. Select $10/mo

- Screenshot if needed:
Screenshot is attached

- Any related issues:

**Comment by bradleyrichter** [comment]
@diracdeltas Could this have anything to do with the coinbase widget flag?

**Comment by mrose17** [comment]
@bradleyrichter - no. it is the same issue (as the more recently reported) https://github.com/brave/browser-laptop/issues/3742

@evoorhees - thanks for reporting this. sorry i didn't see it earlier. i believe it is currently fixed in the master branch. sadly, it's a sequencing error that triggers a `null` dereference. anyway, the easy way to avoid (until you get an image with the fix) is to create the wallet first and then select the contribution amount.

sorry!

fixed in https://github.com/brave/browser-laptop/commit/e165637b066c50f62af7dec961dea4ebf8c55018
